<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyBookView</class>
 <widget class="QWidget" name="ModifyBookView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="viewBackground" stretch="2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="bookInfoLayout">
       <item>
        <layout class="QGridLayout" name="inputsLayout">
         <item row="2" column="1">
          <widget class="QLineEdit" name="isbnLineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="genreLineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="authorLabel">
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="isbnLabel">
           <property name="text">
            <string>ISBN</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QTextEdit" name="descriptionTextEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="titleLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="authorLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="sectionLabel">
           <property name="text">
            <string>Section</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="genreLabel">
           <property name="text">
            <string>Genre</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="sectionLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="coverPreviewLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="text">
          <string>Drag Cover Image Here</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Update Book</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>deleteButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
